examples/graphene_apps/python_workloads/docker-compose.yaml

```diff
@@ -19,8 +19,8 @@ services:
     container_name: process-work-order
     image: avalon-python-worker-dev
     build:
-      context: .
-      dockerfile: ./Dockerfile
+      context: ../../..
+      dockerfile: examples/graphene_apps/python_worker/Dockerfile
```

Comment:
Dockerfile to build Avalon python worker should be kept at same location as Avalon python worker (eg: $TCF_HOME/tc/graphene/python_worker)

Since this is a docker-compose file to unit test Avalon python worker this can also be kept in $TCF_HOME/tc/graphene/python_worker

Reply:
Ok, will move docker-compose.yaml and compose folder with all yaml files to $TCF_HOME/tc/graphene/python_worker